pkgs/development/python-modules/html-sanitizer/default.nix

```diff
@@ -40,15 +43,18 @@ buildPythonPackage rec {
     # Tests are sensitive to output
     "test_billion_laughs"
     "test_10_broken_html"
+
+    # Mismatch snapshot (AssertionError)
+    "test_keep_typographic_whitespace"
```

Comment:
Same here: can we report this failure upstream and link to it here?

Reply:
Done :)